Makefile

```diff
@@ -28,9 +29,9 @@ install:
 	# ensure resolving works inside the chroot
 	cat /etc/resolv.conf > $(DESTDIR)/etc/resolv.conf
 	# copy-in launchpad's build archive
-	if grep -q ftpmaster.internal /etc/apt/sources.list; then \
+	if grep -q ftpmaster.internal /etc/apt/sources.list.d/lp-buildd.sources; then \
```

Comment:
From Noble (24.04) buildd no longer puts sources into `sources.list` but instead truncates this file and writes `/etc/apt/sources.list.d/lp-buildd.sources`